### PR TITLE
docs(add docs for dashboard) - docs for accessing dashboard addon

### PR DESCRIPTION
**Pull request by wallies**

### CODE REVIEW

File: docs/addons/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+After deploying the cluster if you run
```
Comment (enxebre) [inline docs/addons/dashboard.md]: This is only true for AWS

Comment (wallies) [inline docs/addons/dashboard.md]: True. Thus was just a first pass at docs for the addons

**Comment by enxebre** [comment]
need  rebase

**Comment by enxebre** [comment]
I just rebased and fixed some typos

**Pull request merged**